examples/misc/lib/language_tour/generics/misc.dart

```diff
@@ -12,3 +12,9 @@ void miscDeclAnalyzedButNotTested() {
     // #enddocregion collection-literals
   }
 }
+
+// #docregion non-nullable
+class Foo<T extends Object> {
+  // Foo works on objects with the non-nullable type T.
```

Comment:
This is a little off to me.  It's true, but the sentence is also true for `T extends Object?`.  I think you want to say something closer to "// Any type provided to Foo for T must be non-nullable"